frontend/src/components/MainToolbar.vue

```diff
@@ -104,8 +104,17 @@ SPDX-License-Identifier: Apache-2.0
           <v-divider></v-divider>
           <v-card-actions class="px-3">
             <v-icon color="primary" class="ml-2 mr-3">mdi-brightness-6</v-icon>
-            <v-switch v-model="darkMode" dense hide-details class="ma-0" color="primary" @click.native.stop></v-switch>
-            <span class="primary--text text-button">Dark</span>
+            <v-btn-toggle v-model="darkMode" borderless mandatory @click.native.stop>
```

Comment:
add tooltips

Reply:
ah yes I forgot that actually I wanted to do this

Reply:
done